docs/financial/financial-entities.md

```diff
@@ -0,0 +1,61 @@
+This file exists to give an overview of the Financial entities involved in recording Financial data in CiviCRM.
+
+You should only interact with the Financial entities through the relevant apis - 
+generally Order & Payment apis.
+
+It is a work in progress!
```

Comment:
What is a work in progress? If the docs we have a box/template for that used on the other financial pages.